assets/model-diagram.svg

```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" aria-labelledby="chart-title-mermaid-1669067354847 chart-desc-mermaid-1669067354847" style="max-width:396.227px;background-color:#fff;font-family:&quot;trebuchet ms&quot;,verdana,arial,sans-serif;font-size:16px;fill:#333" viewBox="0 0 396.227 815">
+  <defs>
+    <marker id="b" markerHeight="18" markerWidth="18" orient="auto" refX="0" refY="9">
+      <path fill="none" stroke="gray" d="M9 0v18m6-18v18"/>
+    </marker>
+  </defs>
+  <defs>
+    <marker id="a" markerHeight="18" markerWidth="18" orient="auto" refX="18" refY="9">
+      <path fill="none" stroke="gray" d="M3 0v18M9 0v18"/>
+    </marker>
+  </defs>
+  <defs>
+    <marker id="d" markerHeight="36" markerWidth="45" orient="auto" refX="27" refY="18">
+      <path fill="none" stroke="gray" d="M3 9v18m6-9q18-18 36 0-18 18-36 0"/>
+    </marker>
+  </defs>
+  <defs>
+    <marker id="c" markerHeight="36" markerWidth="57" orient="auto" refX="39" refY="18">
+      <circle cx="9" cy="18" r="6" fill="#fff" stroke="gray"/>
+      <path fill="none" stroke="gray" d="M21 18q18-18 36 0-18 18-36 0"/>
+    </marker>
+  </defs>
+  <path fill="none" stroke="gray" marker-end="url(#a)" marker-start="url(#b)" d="M73.762 95v8.333c0 8.334 0 25 10.862 41.667 10.863 16.667 32.588 33.333 43.45 41.667L138.937 195" class="er relationshipLine"/>
+  <path fill="none" stroke="gray" marker-end="url(#a)" marker-start="url(#b)" d="M301.875 95v8.333c0 8.334 0 25-10.863 41.667-10.862 16.667-32.587 33.333-43.45 41.667L236.7 195" class="er relationshipLine"/>
+  <path fill="none" stroke="gray" marker-end="url(#c)" marker-start="url(#b)" d="M187.818 270v100" class="er relationshipLine"/>
+  <path fill="none" stroke="gray" marker-end="url(#a)" marker-start="url(#b)" d="M187.818 445v100" class="er relationshipLine"/>
+  <path fill="none" stroke="gray" marker-end="url(#d)" marker-start="url(#b)" d="M187.818 620v100" class="er relationshipLine"/>
+  <path fill="#f0fff0" fill-opacity="100%" stroke="gray" d="M20 20h107.523v75H20z" class="er entityBox"/>
+  <text class="er entityLabel" dominant-baseline="middle" style="font-family:&quot;trebuchet ms&quot;,verdana,arial,sans-serif;font-size:16px" text-anchor="middle" transform="translate(73.762 57.5)">User agent</text>
```

Comment:
I wonder why these style attributes exist, despite the overall SVG element already having that style. Oh well; I assume there's nothing we can do.

Reply:
https://github.com/whatwg/storage/issues/154